Agreements/Airport/AA_Rat.txt

```diff
@@ -0,0 +1,11 @@
+COPX:EGAA:*:*:STN:*:AAAPP:Rathlin:*:9000:^BEL
+COPX:EGAC:*:*:STN:*:AAAPP:Rathlin:*:9000:^BEL
+
+COPX:EGAA:*:MAC:*:*:AAAPP:Rathlin:*:9000:^BEL
+COPX:EGAC:*:MAC:*:*:AAAPP:Rathlin:*:9000:^BEL
+
+COPX:EGAA:*:DEGOS:*:*:AAAPP:Rathlin:*:9000:^BEL
+COPX:EGAC:*:DEGOS:*:*:AAAPP:Rathlin:*:9000:^BEL
+
+COPX:EGAA:*:MIMKU:*:*:AAAPP:Rathlin:*:9000:^BEL
+COPX:EGAC:*:MIMKU:*:*:AAAPP:Rathlin:*:9000:^BEL
```

Comment:
According to the issue, should be FIXAFTER

Reply:
Fixed